src/views/Setup.vue

```diff
@@ -34,35 +73,104 @@ onMounted(() => {
         </router-link>
       </div>
       <div class="px-4 md:px-0">
-        <h1 v-text="$t('setup.createASpace')" class="mb-4" />
-      </div>
-      <Block>
-        <div class="mb-3">
-          {{ $t('setup.useExistingEns') }}
-        </div>
-        <UiButton class="text-left w-full mb-3 flex px-3">
-          <input
-            v-model="id"
-            class="input flex-auto"
-            :placeholder="$t('setup.example')"
-          />
-          <a
-            class="block py-1 -mr-2"
-            target="_blank"
-            href="https://docs.snapshot.org/spaces/create"
-          >
+        <h1 class="mb-4">
+          {{ $t('setup.createASpace') }}
+          <a target="_blank" href="https://docs.snapshot.org/spaces/create">
             <Icon name="info" size="24" class="text-color p-1" />
           </a>
-        </UiButton>
+        </h1>
+      </div>
+      <div v-if="web3Account" class="px-4 md:px-0">
+        <UiLoading v-if="loadingOwnedEnsDomains" />
```

Comment:
We should use `RowLoading` or `PageLoading` for this loader.

Reply:
Or maybe this load should be put somewhere else. I don't really understand why `loadingOwnedEnsDomains` needs it's own loader before any other UI components are shown.